articles/azure-app-configuration/concept-enable-rbac.md

```diff
@@ -28,9 +28,10 @@ When an RBAC role is assigned to an Azure AD security principal, Azure grants ac
 ## Built-in RBAC roles for Azure App Configuration
 Azure provides the following built-in RBAC roles for authorizing access to App Configuration data using Azure AD and OAuth:
 
-- Azure App Configuration Data Owner: Use this role to give read/write access to App Configuration resources.
-- Azure App Configuration Data Reader: Use this role to give read access to App Configuration resources.
-- Contributor: Use this role to give admin access to the service without granting access to the data stored in the App Configuration instance.
+- Azure App Configuration Data Owner: Use this role to give read/write/delete access to App Configuration data.
+- Azure App Configuration Data Reader: Use this role to give read access to App Configuration data.
+- Contributor: Use this role to give admin access to the App Configuration resource.
```

Comment:
How about adding: "This does not grant access to the data stored in App Configuration."

Reply:
While the contributor role does not grant access to the data via AAD, it does have access to the data using connection strings. Adding this could be confusing.

Reply:
Furthermore, since the contributor role allows to assign any Data roles to the user, the user can even access the data using AAD.

Reply:
Updated to below. I also removed the wording of *admin access* as it can be confusing. Instead, I used the word "manage" from the [official document](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#contributor) of the Contributor role.

Contributor: Use this role to manage the App Configuration resource. While the App Configuration data can be accessed using access keys, this role does not grant access to the data using Azure AD.